docs/image.rst

```diff
@@ -0,0 +1,126 @@
+Image
+*****
+
+.. py:module:: microbit
+
+Classes
+=======
+
+.. py:class::
+    Image(string)
+    Image(width=None, height=None, buffer=None)
+
+    If ``string`` is used, it has to consist of digits 0-9 arranged into
+    lines, describing the image, for example::
+
+        image = Image("""\
+        90009
+        09090
+        00900
+        09090
+        90009""")
```

Comment:
We treat spaces as zeroes. So `"9   9\n 9 9 \n  9  \n 9 9 \n9   9\n"` is a cross.
Also, I think `\n` is not a great line separator. `"90009:09090:00900::09090:90009"` might be better.